src/main/java/com/instinctools/clientaccountmanager/dao/BasicDao.java

```diff
@@ -0,0 +1,18 @@
+package com.instinctools.clientaccountmanager.dao;
+
+import java.util.List;
+
+public interface BasicDao<T> {
+	
+	public T create(Integer userId, String name);
+	
+	public T create(String name);
```

Comment:
Why we need to methods to create something?
it should be something like 
`public T create(T objectToSave)`